src/Microsoft.DotNet.Tools.Publish/Program.cs

```diff
@@ -50,18 +54,28 @@ public static int Main(string[] args)
                     path = Directory.GetCurrentDirectory();
                 }
 
-                // Load project context and publish it
-                var fx = NuGetFramework.Parse(framework.Value());
-                var rids = new[] { runtime.Value() };
-                var context = ProjectContext.Create(path, fx, rids);
+                var projectContexts = ProjectContext.CreateContextForEachTarget(path);
+                projectContexts = GetMatchingProjectContexts(projectContexts, framework.Value(), runtime.Value());
 
-                if (string.IsNullOrEmpty(context.RuntimeIdentifier))
+                if(projectContexts.Count() == 0)
```

Comment:
As one of the primary offenders with this particular code style suggestion, I feel entitled to say: "Spacing" ;P